NuGet.config

```diff
@@ -2,7 +2,7 @@
 <configuration>
     <packageSources>
         <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab" />
-        <add key="docfx-dev" value="https://www.myget.org/F/docfx-dev/api/v3/index.json" />
+        <add key="docs-build-v2-prod" value="https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-build-v2-prod/nuget/v3/index.json" />
```

Comment:
Does this feed corresponds to _dev_ or _master_ branch? We need _dev_ branch feed

Reply:
This feed is for master branch, I'll update to dev branch feed